environments/development.docker-compose.yaml

```diff
@@ -42,13 +42,13 @@ services:
     # Mount the source code of the test project to allow local testing.
     - ../local-test-repo:/app/local-test-repo
     - /var/run/docker.sock:/var/run/docker.sock
-    ports: ["8081:8081"]
+    ports: ["${VITE_PIPELINE_PORT?required}:${VITE_PIPELINE_PORT?required}"]
```

Comment:
See my note on naming, but just to reiterate that this is very confusing: what does vite have to do with the pipeline?